src/models/ratings.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, { Schema } from 'mongoose'
+
+const Rating = mongoose.model(
+    'Rating',
+    new Schema({
+        user: {
+            type: mongoose.Types.ObjectId,
+            ref: 'User',
+            required: true,
+        },
+        sprint: {
+            type: Number,
+            required: true,
+        },
+        quantity: {
+            type: String,
+            required: true,
+        },
+        remark_quantity: {
+            type: String,
+            default: 'no remark'    
+        },
+        quality: {
+            type: String,
+            required: true,
+        },
+        remark_quality: {
+            type: String,
+            default: 'no remark'
+        },
+        professional_Skills: {
+            type: String,
+            required: true,
+        },
+        remark_professional_Skills: {
+            type: String,
+            default: 'no remark'
+        },
+        approved: {
+            type: Boolean,
+            required: true,
```

Comment:
You can not have default and required at the same time since one cancels one out